### PR TITLE
Isolated mode

### PR DESCRIPTION
**Pull request by lightsofapollo**

### CODE REVIEW

File: test/integration/isolated_mode_test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+suite('Shutdown on idle', function() {
+  var co = require('co');
```
Comment (gregarndt) [inline test/integration/isolated_mode_test.js]: for new tests that we write, do we want to still use co or follow babel/es6 conventions?

File: deploy/bin/configure.js [hunk 2]
```diff
@@ -42,10 +42,6 @@ function* configure() {
     currentConfig[key] = (yield prompt(humanDesc)) || defaultValue;
   }
 
-  console.log();
-  console.log(util.inspect(currentConfig, { colors: true }));
-  console.log();
-
   // Yeah bad things will happen if rejected too often...
   if (!(yield prompt.confirm("Does this look right? "))) {
```
Comment (gregarndt) [inline deploy/bin/configure.js]: We removed outputting the contents of the config to confirm if it looks correct, do we want to remove this prompt as well?

Comment (lightsofapollo-taskcluster) [inline deploy/bin/configure.js]: ah!! this is a mistake now that I think about it.

File: test/integration/isolated_mode_test.js (hunk 1 above)
Comment (lightsofapollo-taskcluster) [inline test/integration/isolated_mode_test.js]: your right i am being lazy here :) bad copy/paste!

**Pull request merged**